package.json

```diff
@@ -136,5 +137,8 @@
     ]
   },
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "@prisma/client": "latest"
+  }
```

Comment:
We don't want to make any changes to the apckage.json file. I would say run `git checkout develop -- package.json` so that you get the package.json file from the develop branch